contracts/interfaces/registries/IIPAssetRegistry.sol

```diff
@@ -36,6 +36,25 @@ interface IIPAssetRegistry is IIPAccountRegistry {
         bytes metadata
     );
 
+    // TODO: replace the IPRegistered event with IPRegisteredPermissionless
```

Comment:
Can you provide a rationale for this TODO?

Reply:
`IPRegisteredPermissionless` will eventually be renamed to `IPRegistered` after cleaning up "permissioned" register functions, as the "IPRegistered" event is currently still used by the "permissioned" registering functions.

Reply:
Oh I see, thanks for the clarification